app/assets/stylesheets/_base.scss

```diff
@@ -4,7 +4,7 @@ html{
 }
 
 body{
-  padding: 100px 20px 100px 20px;
+  padding: 20px 20px 100px 20px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `20px 20px 100px` instead of `20px 20px 100px 20px`